assets/sass/patterns/organisms/site-header.scss

```diff
@@ -98,37 +115,21 @@
 @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
 
   .site-header__title {
-    border-right: 1px solid $color-text-dividers;
     float: left;
-    @include height(95);
-    margin-right: 10px;
-    @include margin(10, "right");
-    @include padding(14, "top");
-    @include padding(20, "right");
     position: relative;
-    width: 170px;
+    width: 88px;
   }
 
-  // Stops the top border of .nav-primary from bleeding into .site-header__title.
-  .site-header__title:after {
-    background-color: #fff;
-    content: "";
-    display: block;
-    height: 95px;
-    left: 0;
-    position: absolute;
-    top: 0;
-    width: 169px;
-  }
+  // Stops the top border of .nav-primary from bleeding into site-header__title.
```

Comment:
Surely this comment can go. It seems to correspond to the style that has been removed. `site-header__title:after`